src/main/java/org/jenkinsci/test/acceptance/po/BuildWithParameters.java

```diff
@@ -49,6 +49,11 @@ public BuildWithParameters enter(List<Parameter> definitions, Map<String, ?> val
     }
 
     public void start() {
-        clickButton("Build");
+        if (Boolean.getBoolean("new-job-page.flag.defaultValue")) {
```

Comment:
My fear is that this is ok in testing while the "original" value of the property is false (when the flag value is coming from the default flag value, coming from the BooleanExperimentalFlag class). But when we change a flag default value (from the flag class itself) from false to true, this won't report the correct status of the flag.

This should be good enough for now but in mid-long term, we would need to have some sort of API in ATH to get the flag value.

cc @jtnord @janfaracik